components/dashboard/EmploymentOutlookLauchpad.js

```diff
@@ -28,10 +28,6 @@ const useStyles = makeStyles(theme => ({
   },
 }));
 
-const handleClick = () => {
```

Comment:
It seems were losing this specific event, or are we gaining all clicks after this rolls out?